test/jdk/ProblemList.txt

```diff
@@ -886,5 +886,11 @@ java/awt/print/PrinterJob/ScaledText/ScaledText.java 8231226 macosx-all
 java/awt/font/TextLayout/TestJustification.html 8250791 macosx-all
 javax/swing/JTabbedPane/4209065/bug4209065.java 8251177 macosx-all
 java/awt/TrayIcon/DragEventSource/DragEventSource.java 8252242 macosx-all
+java/awt/FileDialog/DefaultFocusOwner/DefaultFocusOwner.java 7187728 macosx-all,linux-all,solaris-all
```

Comment:
Al usage of Solaris-xxx in the problem list was removed by the https://github.com/openjdk/jdk/commit/071bd521bca2485c6df95c45e4310d39b05bd046#diff-80063aaf72a19fa27f001c17fc7687086ea71b35b01f5e2b3fcd7f50d8c682fbL252